src/App/Pages/Home.elm

```diff
@@ -15,22 +15,22 @@ import List
 import List.Extra
 
 
-view : Model -> Html Msg
-view model =
+view : Model -> WebData (List Article) -> WebData (List Recipe) -> WebData (List Recipe) -> Html Msg
+view model promotedArticles promotedRecipes recipes =
     div []
-        [ viewPromotedContent model
+        [ viewPromotedContent promotedArticles promotedRecipes
         , viewCurrentMonthIssue model
         , viewCookMenu model
-        , viewRecipes model
+        , viewRecipes recipes
         ]
 
 
-viewPromotedContent : Model -> Html Msg
-viewPromotedContent model =
+viewPromotedContent : WebData (List Article) -> WebData (List Recipe) -> Html Msg
```

Comment:
Oh, and just to be extra strict. http://package.elm-lang.org/packages/eeue56/elm-html-test/latest , so the other JS frameworks will envy :)